test/controllers/renewal/CashPaymentsCustomersNotMetControllerSpec.scala

```diff
@@ -55,32 +53,40 @@ class ReceiveCashPaymentsControllerSpec extends AmlsSpec with MockitoSugar {
       .thenReturn(Future.successful(new CacheMap("", Map.empty)))
   }
 
-  "ReceiveCashPaymentsController" when {
-
+  "CashPaymentsCustomersNotMet" when {
     "get is called" must {
       "load the page" when {
         "renewal data is found for receiving payments and pre-populate the data" in new Fixture {
-
           when(mockRenewalService.getRenewal(any(),any(),any()))
             .thenReturn(Future.successful(Some(Renewal(receiveCashPayments = Some(receiveCashPayments)))))
 
           val result = controller.get()(request)
           status(result) mustEqual OK
 
           val page = Jsoup.parse(contentAsString(result))
-          page.select("input[type=radio][name=receivePayments][value=true]").hasAttr("checked") must be(true)
-          page.select("input[type=radio][name=receivePayments][value=false]").hasAttr("checked") must be(false)
+          page.select("input[type=radio][name=receiveCashPayments][value=true]").hasAttr("checked") must be(true)
+          page.select("input[type=radio][name=receiveCashPayments][value=false]").hasAttr("checked") must be(false)
+        }
+
+        "renewal data is found for not receiving payments and pre-populate the data" in new Fixture {
```

Comment:
I am struggling to make sense of this sentence a bit. 

Is

`receive cash payments is no, renewal data is found and render form`

Better?  Maybe consider rewording as I had to read this a few times.

Reply:
you're totally right I'll change that